app/presenters/tree_builder_pxe_customization_templates.rb

```diff
@@ -11,8 +11,7 @@ def set_locals_for_render
   end
 
   def root_options
-    title = _("All %{template} - %{type}") % {:template => ui_lookup(:models => 'CustomizationTemplate'),
-                                              :type     => ui_lookup(:models => 'PxeImageType')}
+    title = _("All Customization Templates - System Image Types")
```

Comment:
Only thing that this approach loses is a centralized place to name the models in English.  The translation is being done one the fly - so that, in the future, if we decide that the `PxeImageType` should be renamed from `System Image Types` to `New and Improved System Image Types`, we will have to make this update in multiple places.  /cc @Fryguy